solutions/src/main/java/ru/sberbank/school/task02/FxRequestConverter.java

```diff
@@ -0,0 +1,60 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.exception.FxConversionException;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.FxRequest;
+import ru.sberbank.school.task02.util.Symbol;
+
+import java.math.BigDecimal;
+import java.util.Objects;
+
+/**
+ * Created by Mart
+ * 10.05.2019
+ **/
+public class FxRequestConverter {
+
+    public static ClientOperation getClientOperation(FxRequest request) {
+        Objects.requireNonNull(request);
+        ClientOperation clientOperation = null;
+        switch (request.getDirection()) {
+            case "BUY":
+                clientOperation = ClientOperation.BUY;
+                break;
+            case "SELL":
+                clientOperation = ClientOperation.SELL;
+                break;
+        }
+        return clientOperation;
+    }
+
+    public static Symbol getSymbol(FxRequest request) {
+        Objects.requireNonNull(request);
+        Symbol symbol = null;
+        switch (request.getSymbol()) {
+            case "USD/RUB":
+                symbol = Symbol.USD_RUB;
+                break;
+            case "RUB/USD":
+                symbol = Symbol.RUB_USD;
+                break;
+        }
+        return symbol;
+    }
+
+    public static BigDecimal getAmount(FxRequest request) {
+        Objects.requireNonNull(request);
+        BigDecimal amount;
+        try {
+            amount = new BigDecimal(request.getAmount());
+            if (amount.compareTo(BigDecimal.ZERO) <= 0) {
+                throw new FxConversionException("Wrong input amount of money:"
+                        + request.getAmount() + " expected amount > 0");
+            }
+            return amount;
+        }catch(NumberFormatException e) {
```

Comment:
WhitespaceAround: &#39;}&#39; is not followed by whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-2017926729>*